compute_tools/src/compute.rs

```diff
@@ -1064,19 +1063,25 @@ impl ComputeNode {
                     }
 
                     let conf = Arc::new(conf);
+                    let mut phases = vec![
+                        DeleteDBRoleReferences,
+                        ChangeSchemaPerms,
+                        HandleAnonExtension,
+                    ];
+
+                    if spec.drop_subscriptions_before_start {
+                        info!("Adding DropLogicalSubscriptions phase because drop_subscriptions_before_start is set");
```

Comment:
NIT: all other logs start with lowercase
```suggestion
                        info!("adding DropLogicalSubscriptions phase because drop_subscriptions_before_start is set");
```